e2e-test-audit.csv

```diff
@@ -0,0 +1,292 @@
+Command,E2E-Atlas,E2E-OM,E2E-CM,Deprecated,Atlas,OM,CM,Location,E2E Test Location

```

Comment:
[q] what are you plans on maintaining this and consider that this is a public repo and we welcome external contributors, this format seems hard to maintain specially when adding new commands?

Reply:
Given that this file is a `.csv`, it can be opened with any editor and easily updated

Reply:
I think the point here is not that it's easy to update, but that it's hard to know what update is needed. Think even if you look at the format in 6 months, will you find it easy to remember what each field does?

If we were using a file input like the above, a son would be more powerful as it can be more expressive and easier to validate.

Reply:
even as a `.csv` simple text editors (VS code/Atom/Sublime) may show this in a not tabular (excel) way. I would say [markdown table](https://guides.github.com/features/mastering-markdown/#GitHub-flavored-markdown) could be a better presentation as you could preview the file in your text editor and not need something like excel to make sure you didn't break it

this also goes with my other comment on percentages, I doubt people will take care to update them, so should we keep them? 

I'd like to keep this requirement for an e2e inventory as simple and welcoming for someone not familiar with the codebase as possible so if we need to make changes to how we structure e2e tests to make this inventory even easier I'm ok with that